content/work/adobe.md

```diff
@@ -218,10 +218,10 @@ SoundWorks]. Adding sound elevated the whole experience of game play.
 ### CSS Custom Properties
 
 Each question in the game has nine options, and each option consists of the same
-identical image plus a unique font. Instead of using a seperate image for each
-item on the page, we wanted to give Adobe more flexibility to easily edit or
-add new font options and questions.
-We used a single SVG, and controlled the text and fonts via HTML/CSS.
+identical image plus a unique font. Instead of using a separate image for each
```

Comment:
dang it, I had a note to self about separate vs seperate... forgot to check. That is the one word that ALWAYS trips me up.